pkg/svcat/service-catalog/broker.go

```diff
@@ -163,22 +173,43 @@ func (sdk *SDK) Register(brokerName string, url string, opts *RegisterOptions) (
 }
 
 // Sync or relist a broker to refresh its catalog metadata.
-func (sdk *SDK) Sync(name string, retries int) error {
+func (sdk *SDK) Sync(name string, scopeOpts ScopeOptions, retries int) error {
 	for j := 0; j < retries; j++ {
-		catalog, err := sdk.RetrieveBroker(name)
-		if err != nil {
-			return err
-		}
+		if scopeOpts.Scope.Matches(ClusterScope) {
+			catalog, err := sdk.RetrieveBroker(name)
+			if err != nil {
+				return err
+			}
 
-		catalog.Spec.RelistRequests = catalog.Spec.RelistRequests + 1
+			catalog.Spec.RelistRequests = catalog.Spec.RelistRequests + 1
 
-		_, err = sdk.ServiceCatalog().ClusterServiceBrokers().Update(catalog)
-		if err == nil {
-			return nil
+			_, err = sdk.ServiceCatalog().ClusterServiceBrokers().Update(catalog)
+			if err == nil {
+				return nil
+			}
+			if !errors.IsConflict(err) {
+				return fmt.Errorf("could not sync service broker (%s)", err)
+			}
 		}
-		if !errors.IsConflict(err) {
-			return fmt.Errorf("could not sync service broker (%s)", err)
+
+		if scopeOpts.Scope.Matches(NamespaceScope) {
+			namespace := scopeOpts.Namespace
+			catalog, err := sdk.RetrieveNamespaceBroker(namespace, name)
```

Comment:
Can you rename `catalog` to `broker` here and in the original code?

Reply:
done